Command/UpdateCoreRoutesCommand.php

```diff
@@ -31,7 +31,7 @@ public function configure()
     {
         $this->setName('sonata:page:update-core-routes');
         $this->setDescription('Update core routes, from routing files to page manager');
-        $this->addOption('all', null, InputOption::VALUE_NONE, 'Create snapshots for all sites');
```

Comment:
Do you think people might run this command with this option in a cron job? If yes, it is a BC-break IMO.